go.mod

```diff
@@ -1,22 +1,22 @@
 module github.com/segmentio/chamber/v2
 
-go 1.19
```

Comment:
do we need to bump this as well?

IIRC, we hesitated on bumping this because of breaks to packages that pull in `chamber` as a dependency.

I would vote to keep this on 1.19 now since this represents a _minimum_ version for pulling in

Reply:
Ah, you're right

Reply:
This is fixed